services/models.py

```diff
@@ -0,0 +1,33 @@
+from django.db import models
+
+class Review(models.Model):
+    user         = models.ForeignKey('users.User',on_delete=models.CASCADE,related_name='review')
+    product      = models.ForeignKey('products.Product',on_delete=models.CASCADE,related_name='review')
+    order_item   = models.ForeignKey('orders.OrderItem',on_delete=models.CASCADE,related_name='review')
```

Comment:
review가 order_item을 가지고 있는 이유가 뭔가요?

```suggestion
    order_item   = models.ForeignKey('orders.OrderItem',on_delete=models.CASCADE,related_name='review')
```

Reply:
구매 이력이 있어야 한다고 해서 넣었는데 생각해보니까 view에서 if문으로 처리 가능할 것 같습니다